code/modules/mob/mob_helpers.dm

```diff
@@ -104,6 +104,10 @@ mob/proc/remove_internal_organ()
 			return 1
 	return 0
 
+/proc/ghostCheck(A) //Ghostbusters. Returns 1 if it's a ghost, returns 0 if they're an adminghost or otherwise
+	if(isobserver(A) && !isAdminGhost(A))
+		return 1
+	return 0
```

Comment:
This is not necessary. 0 is the default return value.

Reply:
Strictly speaking, the default return value is `null`
But close enough